### ISSUE TITLE
Add Sublime Text Theme

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [ajaxorg/ace-builds#167]

**Comment by nightwing** [comment]
Do you mean some specific theme named "Sublime Text" or generall support for sublime text theme format? If the former could you show the repository of the specific theme you want.

**Comment by Abdur-rahmaanJ** [comment]
@nightwing yes a specific theme named "Sublime Text" which colours things as sublime text does

**Comment by nightwing** [comment]
But sublime text itself doesn't have a color scheme named "Sublime Text" it uses the Monokai theme by default, which is supported in ace too.

**Comment by Abdur-rahmaanJ** [comment]
@nightwing there is some difference between sublime's interpretation of monokai which makes sublime's monokai a theme of it's own
